zsl/constants.py

```diff
@@ -0,0 +1,8 @@
+class MimeType(object):
+    APPLICATION_JSON = 'application/json'
```

Comment:
Yes, I tried to use it, but it has the .value thing which

1. I was guessing that you'll have problems with constants, which you did have and 
2. if there's the .value thing added, then there's much bigger problem.

fixed.